config/parser.go

```diff
@@ -184,6 +184,8 @@ func (p *Parser) parseLines(lines []string) (err error) {
 			p.opts.metricsRefreshInterval = parseInt(value, defaultMetricsRefreshInterval)
 		case "METRICS_ALLOWED_NETWORKS":
 			p.opts.metricsAllowedNetworks = parseStringList(value, []string{defaultMetricsAllowedNetworks})
+		case "USER_AGENT":
```

Comment:
Can you please add the prefix `HTTP_CLIENT` (ie `HTTP_CLIENT_USER_AGENT`) to be consistent with other HTTP client options.

Reply:
Yup!